### PR TITLE
respect --image flag value for service catalog image name

### PR DESCRIPTION
**Pull request by bparees**
bug 1475219
https://bugzilla.redhat.com/show_bug.cgi?id=1475219

### CODE REVIEW

**Comment by bparees** [comment]
@csrwng ptal

File: examples/service-catalog/service-catalog.yaml [hunk 1]
```diff
@@ -312,7 +312,7 @@ objects:
           - service-catalog
           - --broker-relist-interval
           - "5m"
-          image: openshift/origin-service-catalog:${SERVICE_CATALOG_TAG}
+          image: ${SERVICE_CATALOG_NAME}
```
Comment (csrwng) [inline examples/service-catalog/service-catalog.yaml]: s/SERVICE_CATALOG_NAME/SERVICE_CATALOG_IMAGE/

File: examples/service-catalog/service-catalog.yaml [hunk 2]
```diff
@@ -356,11 +356,11 @@ parameters:
   name: CORS_ALLOWED_ORIGIN
   required: true
   value: 10.192.213.116
-- description: Tag of the service catalog images to use for apiserver and controller-manager
-  displayName: Service catalog image tag
-  name: SERVICE_CATALOG_TAG
+- description: Name of the service catalog images to use for apiserver and controller-manager
```
Comment (csrwng) [inline examples/service-catalog/service-catalog.yaml]: s/images/image/

File: examples/service-catalog/service-catalog.yaml (hunk 1 above)
Comment (bparees) [inline examples/service-catalog/service-catalog.yaml]: fixed

File: examples/service-catalog/service-catalog.yaml (hunk 2 above)
Comment (bparees) [inline examples/service-catalog/service-catalog.yaml]: fixed

**Comment by bparees** [comment]
@csrwng updated.

**Comment by bparees** [comment]
backport to 3.6: https://github.com/openshift/origin/pull/15502

**Pull request merged**